pages/views/_includes/navigation/sidebar-item.njk

```diff
@@ -44,7 +45,7 @@
               <sup class="badge badge-sup badge-warning badge-sidebar">beta</sup>
             {% endif %}
             {% if isDraft %}
-              <sup class="badge badge-sup badge-danger badge-sidebar">draft</sup>
+              <sup class="badge badge-sup badge-danger badge-sidebar {{ 'badge-hidden' if isDraftCollection }}">draft</sup>
```

Comment:
mb instead of adding this class you can use `hidden` attribute? since we already have a style for this attribute
```
[hidden] {
  display: none !important;
}
```